Doc/library/sqlite3.rst

```diff
@@ -222,14 +222,16 @@ Module functions
 
 .. function:: complete_statement(statement)
 
-   Returns ``True`` if the string *statement* contains one or more complete SQL
-   statements terminated by semicolons. It does not verify that the SQL is
-   syntactically correct, only that there are no unclosed string literals and the
-   statement is terminated by a semicolon.
+   Return ``True`` if the string *statement* appears to contain one or more
+   complete SQL statements terminated by semicolons.
+   No syntactic verification or parsing of any kind is performed.
 
-   This can be used to build a shell for SQLite, as in the following example:
+   This function may be useful during command-line input
+   to determine if the entered text seems to form a complete SQL statement,
+   or if additional input is needed before :meth:`executing <Cursor.execute>`
```

Comment:
It currently displays rather strangely because `executing` is rendered as a literal, which is not what is intended (as it is not). To work around this, I suggest (with `()`)

```suggestion
   or if additional input is needed before :meth:`~Cursor.execute`-ing
```

or (without `()`)

```suggestion
   or if additional input is needed before :meth:`execute <Cursor.execute>`-ing
```